Remotion/Web/Development.WebTesting/ControlSelection/ISingleControlSelector.cs

```diff
@@ -32,16 +31,16 @@ public interface ISingleControlSelector<out TControlObject> : IControlSelector
     /// Selects the only control within the given <paramref name="context"/> matching the type <typeparamref name="TControlObject"/>.
     /// </summary>
     /// <returns>The <see cref="ControlObject"/> for the selected control.</returns>
-    /// <exception cref="AmbiguousException">If multiple controls of type <typeparamref name="TControlObject"/> are found.</exception>
-    /// <exception cref="MissingHtmlException">If no matching control can be found.</exception>
+    /// <exception cref="WebTestException">If multiple controls of type <typeparamref name="TControlObject"/> are found.</exception>

```

Comment:
We should probably merge this into a single exception doc

Reply:
We found docs using this style.